udata/core/followers/factories.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from factory.mongoengine import MongoEngineFactory
+
+from udata import models
+
+
+class FollowFactory(MongoEngineFactory):
```

Comment:
Given the fact that it needs real objects that you give as parameters and that the factory does not provide any attribute, it might be better to not use a factory.